src/observer.rs

```diff
@@ -116,6 +116,14 @@ impl Observer {
     pub fn xyz(&self, light: &dyn Light, filter: Option<&dyn Filter>) -> XYZ {
         self.data().xyz(light, filter)
     }
+
+    pub fn rgb2xyz(&self, rgbspace: RgbSpace) -> Matrix3<f64> {
+        self.data().rgb2xyz(rgbspace)
+    }
+
+    pub fn xyz2rgb(&self, rgbspace: RgbSpace) -> Matrix3<f64> {
```

Comment:
I think we should avoid duplicating the `ObserverData` API on the `Observer` type. It's a lot of manual duplication of API and just increase the public API surface, making it more confusing for a user IMO. There becomes multiple ways of doing the same thing, and not any real clear answer to which one is preferred or how they differ.

Personally I would love if we could hide either `Observer` or `ObserverData` completely from the public API and just have observer be one thing instead of two.